stronk/models/user.py

```diff
@@ -83,7 +88,8 @@ def update(self, attrs):
         if 'username' in attrs:
             self.username = attrs.get('username')
         if 'current_program' in attrs:
-            self.current_program = attrs.get('current_program')
+            new_program = attrs.get('current_program')
+            self.current_program = None if new_program == -1 else new_program
```

Comment:
I realized I forgot to remove this during my last pr but we should only be able to update a user's program via subscribe mutation in program. Could the unsubscribe functionality be moved to the `SubscribeToProgram` mutation for `Program`? :pray:

Reply:
As per discussion, we're going to split this up into another PR.